pandas/tests/test_algos.py

```diff
@@ -11,8 +11,9 @@
 
 from pandas._libs import (
     algos as libalgos, groupby as libgroupby, hashtable as ht)
-from pandas.compat import PY2, lrange, range
-from pandas.compat.numpy import np_array_datetime64_compat
+from pandas.compat import lrange, range
+from pandas.compat.numpy import (
```

Comment:
import the the pandas namespace (for consistency with the rest of the codebase)

I don't remember if np_array_datetime64_compat is from there as well (and I think we are removing this after the numpy bump anyhow)